packages/wrangler/src/d1/migrations/list.tsx

```diff
@@ -16,7 +16,17 @@ import type { BaseSqlExecuteArgs } from "../execute";
 import type { Argv } from "yargs";
 
 export function ListOptions(yargs: Argv): Argv<BaseSqlExecuteArgs> {
-	return Database(yargs);
+	return Database(yargs)
+		.option("local", {
+			describe:
+				"Execute commands/files against a local DB for use with wrangler dev",
+			type: "boolean",
+		})
+		.option("persist-to", {
+			describe: "Specify directory to use for local persistence (for --local)",
+			type: "string",
+			requiresArg: true,
+		});
```

Comment:
Is there any way these options can be unified? Specifying them twice seems a bit redundant

Reply:
I could probably define them centrally (like we do with `Database`) and export

Reply:
Resolving pending followup